<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enroll App</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lblStatus">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>STATUS</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="videoLabel">
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>240</height>
       </size>
      </property>
      <property name="text">
       <string>YOUR_FACE</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEnroll">
      <property name="text">
       <string>등록</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblMessage">
      <property name="text">
       <string>RESULT</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>788</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
